### ISSUE TITLE
Trouble dismissing the modal when outside Pressable component is pressed on web.  

### ISSUE THREAD

**Issue by cmaycumber** [intergalacticspacehighway/react-native-popper#12]
I'm running into some trouble when trying to dismiss the modal when a pressable is pressed outside of the modal on web. The modal currently works as expected when pressing outside of the popper if the element isn't another pressable, however when pressing another pressable e.i a button, the popper stays open. 

Any guidance on this would be great.

**Comment by intergalacticspacehighway** [comment]
@cmaycumber Can you take a look at this example and let me know if this is what you're trying to achieve?
https://codesandbox.io/s/popover-outside-pressable-dismiss-1gko4?file=/src/App.js

**Comment by cmaycumber** [comment]
I tweaked the code slightly to get the desired effect I'm looking for:

```jsx
import * as React from "react";
import { StyleSheet, Pressable, Text, View } from "react-native";
import { Popover } from "react-native-popper";

export default function App() {
  const [isOpen, setIsOpen] = React.useState(false);
  return (
    <View style={styles.wrapper}>
      <Popover
        offset={10}
        trigger={
          <Pressable
            style={{
              paddingHorizontal: 10,
              paddingVertical: 4,
              backgroundColor: "#e3e3e3"
            }}
          >
            <Text>Press here</Text>
          </Pressable>
        }
      >
        <Popover.Content>
          <View style={styles.tooltip}>
            <Text style={styles.tooltipText}>Hello world </Text>
          </View>
        </Popover.Content>
      </Popover>
      <Pressable style={{ marginTop: 200 }}>Close popover</Pressable>
    </View>
  );
}

const styles = StyleSheet.create({
  wrapper: {
    flex: 1,
    alignItems: "center",
    justifyContent: "center"
  },
  tooltipText: {
    color: "#fff"
  },
  tooltip: {
    paddingVertical: 4,
    paddingHorizontal: 8,
    backgroundColor: "#616161e6",
    borderRadius: 4
  }
});
```

I removed most of the props and the action on the pressable.  But clicking on the pressable "Close popover" doesn't close the  popover when the prop `shouldCloseOnOutsideClick` is set to `true`.

**Comment by intergalacticspacehighway** [comment]
Thanks for reporting @cmaycumber :). I just pushed a fix in 0.3.1 and tested your example here. https://codesandbox.io/s/popover-outside-pressable-dismiss-fixed-7h1s4?file=/src/App.js
